test/validate/validate.go

```diff
@@ -154,77 +167,121 @@ func (v *Validator) validateIPs(ctx context.Context, stateFileIps stateFileIpsFu
 		if err != nil {
 			return errors.Wrapf(err, "failed to exec into privileged pod - %s", podName)
 		}
-		filePodIps, err := stateFileIps(result)
+		cnsPodIPs, err := stateFileIps(result)
```

Comment:
Why `cnsPodIps` ? I think this contains all podsIps ?

Reply:
That's true, I guess what I meant to say here, was "pods that CNS is managing"